code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -433,6 +434,10 @@
 			S.switch_mode_to(TRACK_INFILTRATOR)
 		countdown.start()
 		set_security_level(SEC_LEVEL_DELTA)
+
+		if (proper_bomb) // Why does this exist
+			countdown_music = play_soundtrack_music('sound/soundtrack/countdown.ogg', specific_z = SSmapping.levels_by_trait(ZTRAIT_STATION)[1])
```

Comment:
This should play when there is 90 seconds left rather than when its triggered.
Also `SSmapping.levels_by_trait(ZTRAIT_STATION)[1]` is kind of bad. Either make the specific_z thing to check for z-traits instead or make it global.

Reply:
Shouldnt matter if its not a specific Z, since exploration bombs don't count as proper_bombs

Reply:
I don't want mobs who aren't on the station level to hear it

Reply:
What about multi-z maps